commit a0f4288e0b952830d38151264c0a8b042e32d8a4
Author: MatÄ›j Laitl <matej@laitl.cz>
Date:   Tue Jun 26 00:42:50 2012 +0200

    Fix Context view not being fully formed on startup
    
    Something from inside Plasma tries to resize context view's sceneRect()
    to 200x200 px. I don't know why this happens, but let's hope we'll have
    QML context view soon, so don't investigate and use a work-around.
    
    BUG: 292895
    FIXED-IN: 2.6
    DIGEST: popular bug fixed in Amarok

diff --git a/src/context/containments/verticallayout/VerticalToolbarContainment.cpp b/src/context/containments/verticallayout/VerticalToolbarContainment.cpp
index a23305f..8fa6f9b 100644
--- a/src/context/containments/verticallayout/VerticalToolbarContainment.cpp
+++ b/src/context/containments/verticallayout/VerticalToolbarContainment.cpp
@@ -129,7 +129,8 @@ Context::VerticalToolbarContainment::updateGeometry()
     if(!view())
         return;
 
-    QRectF rect = view()->sceneRect();
+    // mimic ContextView::resizeEvent(), nothing else seems to work, bug 292895
+    QRectF rect( view()->pos(), view()->maximumViewportSize() );
     setGeometry( rect );
     m_applets->setGeometry( rect );
     m_applets->refresh();
